syntax = "proto3";

message UpdateResponse {
    enum ResponseType {
        SUCCESS = 0;
        ERROR = 1;
    }
    ResponseType status = 1;
}

message Response {
    string name = 1;
}